PUBLISHING.md

```diff
@@ -0,0 +1,13 @@
+# Publishing Instructions
+
+To update the version number, edit the version in `package.json`. 
+
+To publish a new package through NPM, run the following commands from the root folder: 
+
+```
+yarn build 
+cd dist
+npm publish
```

Comment:
```suggestion
npm publish --tag <alpha | beta>
```

Reply:
I'm going to apply this change to all other open PRs right now.